README.md

```diff
@@ -1,3 +1,51 @@
-# Cocos Creator 3D
-
-TO BE DONE
+# Cocos Creator 3D
+
+Cocos Creator 3D is a complete package of game development tools and workflow, including a game engine, resource management, scene editing, game preview, debug and publish one project to multiple platforms. Based on Cocos Creator,it has been aggresively refactored to meet the complex needs of 3D projects while retaining most of the original features where applicable.

```

Comment:
Cocos Creator 3D is the new generation of game development tool in Cocos family, it brings a complete set of 3d features and provides an intuitive, low cost and collaboration friendly workflow to game developers.

Cocos Creator 3D inherited many good qualities and cool features from its previous versions, such as cross-platform support including instant gaming platforms like WeChat mini game, asset management, powerful animation editing, etc. Furthermore, Cocos Creator 3D has pushed our technology to a whole new level. Our GFX implementation is designed to adapt to the latest graphic backend APIs, it already supports WebGL 2 and can be seamlessly fall back to WebGL 1, it will support Vulkan and Metal in the native build. The material system is built on our own effect format which uses GLSL 300 and can be easily adapted to lower version on low end devices. Developers can produce high performance, extremely expressive shaders with good compatibility. Along with physical based camera and lighting, high level game graphic can be easily achieved. Our pure GPU driven skeleton animation also make sure your game runs as smooth as possible. Besides all these exciting features, Cocos Creator 3D have builtin physics support, terrain editing support, visual effect editing, ui system, TypeScript support, instant preview etc.